man/usr/share/man/man8/piadm.8.md

```diff
@@ -326,6 +326,19 @@ piadm(8) -- Manage SmartOS Platform Images
  [root@smartos ~]#
 ```
 
+## ENVIRONMENT VARIABLES
+
+    The following environment variables affect the execution of PIADM(8).
+
+    PIADM_CHECKSUM_URL      If set, the value will be used to retrieve the md5sums.txt file, which will
```

Comment:
NIT.  If/when we allow other hash algorithms, `md5sums.txt` is probably not gonna be the name.